tools/harness-thci/OpenThread.py

```diff
@@ -1323,8 +1335,14 @@ def ping(self, strDestination, ilength=0, hop_limit=64, timeout=5):
             hop_limit: hop limit
 
         """
-        cmd = 'ping %s %s 1 1 %d %d' % (strDestination, str(ilength), hop_limit, timeout)
+        cmd = 'ping %s %s' % (strDestination, str(ilength))
+        if not self.IsReference20200818:
+            cmd += ' 1 1 %d %d' % (hop_limit, timeout)
+
         self.__executeCommand(cmd)
+        if self.IsReference20200818:
+            # wait echo reply
+            self.sleep(6)  # increase delay temporarily (+5s) to remedy TH's delay updates
```

Comment:
I do not know. Actually I copied this line of code from `OTN.py` in Harness, which I think is intended to work for Thread references.

Reply:
> Are we adding the delay because ping of 202008 outputs Done immediately?

@simonlingoogle  Yes. I have tested and confirmed that it outputs `Done` immediately after `ping`.